DBMerger/Merger.cs

```diff
@@ -301,7 +307,7 @@ private void MergeUsers()
                     }
 
                     var oldAvatarTime = old.Length >= 3 ? (int)old[2] : 0;
```

Comment:
didn't realize avatar time could be null. could you also make sure this access of the row is not null? something like: `((int)old[2]) ?? 0` instead of `(int)old[2]` would work

Reply:
Probably because NullPointerException is raised at `((int)old[2])`, so `((int)old[2]) ? 0` would not solve the problem.
If the condition `old[2] ! = null to work around this.

Reply:
Ahh you're right, good catch. I'm low on sleep lol